### PR TITLE
[Focus Mode] Images and content for the security tool

### PR DESCRIPTION
**Pull request by captainbrosset**
New Focus Mode images and content for the **Understand security issues using the Security tool** article.

Before: https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/security/
After: https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/security/index?branch=pr-en-us-2939

AB#47818857

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/security/index.md [hunk 1]
```diff
@@ -57,29 +57,33 @@ If you haven't set up HTTPS on your server, [Let's Encrypt](https://letsencrypt.
...
+![Mixed content](./index-images/overview-mixed-secure.png)
 
-In the previous figure, select **View 1 request in Network panel** to open the **Network** tool and apply the `mixed-content:displayed` filter so that the **Network Log** only shows non-secure resources.
-
-![Mixed resources in the Network Log](./index-images/security-network-filter.png)
+In Microsoft Edge, mixed content requests are automatically upgraded to HTTPS when possible. If the resource is not available over HTTPS, the request is blocked.
 
 
 <!-- ====================================================================== -->
 ## View details
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/security/index.md:68]: i consider non-blocking and arguably out of scope for this PR:
* h2 followed by h3 without any sentence in between.
* not applied h2/h4/h6 pattern (d/k whole page outline of headings to see if can apply)

File: microsoft-edge/devtools-guide-chromium/security/index.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Use the **Security** panel to inspect the security of a page.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/security/index.md:44]: i consider non-blocking and arguably out of scope for this PR:
* h2 ("Common problems") followed by h3 without any sentence in between.
* not applied h2/h4/h6 pattern (d/k whole page outline of headings to see if can apply)

rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/security/?branch=pr-en-us-2939#common-problems

File: microsoft-edge/devtools-guide-chromium/security/index.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/security/index.md:1]: imo, 3 pixels width of red box line would look better than 5 pixels.   
5 = chunky, disproportionately thick.  
3 = subtle and best for an overlay of UI.

**Review by mikehoffms — APPROVED** [review]
Writer/Editor pass.  
* Requested red boxes 3 px instead of 5 px width of red border line.  
* Noted existing lack of text between h2s & h3s, against style guides (out of scope b/c not text or pngs about F.M.)

**Comment by captainbrosset** [comment]
> * Requested red boxes 3 px instead of 5 px width of red border line.

Done

> * Noted existing lack of text between h2s & h3s, against style guides (out of scope b/c not text or pngs about F.M.)

I converted h3s to h4s, and added intro text beneath h2s.

**Pull request merged**